Makefile

```diff
@@ -0,0 +1,44 @@
+BUILDER=genesis-builder
+CONTAINER=genesis
+# Run QEMU connecting VM ttyS1 to stdio
+QEMU=qemu-system-x86_64 --enable-kvm -smp 2 -m 2048 -kernel output/genesis-vmlinuz -initrd output/genesis-initrd.img -serial null -serial stdio -monitor null
+BASE_ARGS=rootflags=loop root=live:/genesis.iso rootfstype=auto ro vga=791 rd_NO_LUKS rd_NO_MD rd_NO_DM GENESIS_MODE=util console=tty1
+
+.PHONY: help clean run-serial run-serial-notail run-tty1 run-tty1-notail
+
+help:
+	@echo 'Did you mean "make output"?'
+
+docker-image:
+	sudo docker build -t $(BUILDER) .
+	touch docker-image
+
+output: docker-image
+	mkdir $@
+	sudo docker run --name $(CONTAINER) --privileged=true -v $(PWD)/$@:/output $(BUILDER)
+	sudo chown -R $(USER) $@
+
+clean:
+	sudo losetup -a | cut -d: -f1 | xargs -r -n 1 sudo losetup -d
+	sudo rm -rf output docker-image
+	sudo docker rm $(CONTAINER) || true
+	sudo docker rmi $(BUILDER) || true
+
+run-serial: output
```

Comment:
I think the test targets should all be prefixed by `test-` so its clear what their intent is (clearer than reading comments in the makefile)